FirebaseMessaging/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# 2020-06 -- v4.4.2
+- [changed] Use UNNotificationRequest to schedule local notification for local timezone notification for iOS 10 ang above. (#5667)
```

Comment:
Should fixing the problem with the percent character be mentioned?

Reply:
Done.